manifests/0000_90-kube-apiserver-operator_06_alerts-cpu.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: cpu-utilization
+  namespace: openshift-kube-apiserver
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
+spec:
+  groups:
+    - name: master-cpu-utilization
+      rules:
+        - alert: HighOverallMasterCPU
+          summary: >
+            CPU utilization across all three masters is higher than two masters can sustain; a single master outage may
+            cause a cascading failure; increase available CPU.
+          message: >
+            Given three masters, the overall CPU utilization may only only be about 2/3 of all available capacity.
+            This is because if a single master fails, the remaining two must handle the load of the cluster in order to be HA.
+            If the cluster is using more than 2/3 of all capacity, if one master fails, the remaining two are likely to
+            fail when they take the load.
+            To fix this, increase the CPU and memory on your masters.
+          expr: |
+            sum(
+              100 - (avg by (instance) (rate(node_cpu_seconds_total{mode="idle"}[5m])) * 100)
+              AND on (instance) label_replace( kube_node_role{role="master"}, "instance", "$1", "node", "(.+)" )
+            )
+            /
+            count(kube_node_role{role="master"})
+            > 60
+          for: 10m
```

Comment:
- during a kube-apiserver roll out, it is natural for one instance to get more cpu usage, the alert may fire as a `warning` and on a somewhat busy cluster and that is fine, I assume.
- (from the incident we have seen) the cpu spikes briefly and the kube-apiserver crashes, so `5m` average may smooth out these spikes, is it feasible to do `1m` average to take into account those brief spikes. may be someone with monitoring expertise can answer this question.

this looks good to me